raiden/api/validations/valid_light_client.py

```diff
@@ -0,0 +1,29 @@
+from http import HTTPStatus
```

Comment:
I think light_client_authorization its better as file name

valid_light_client is too wide to me, what is a _valid light client_? 

i think the functions here are releted only with api-key/authorization stuff

Reply:
Agree!